core/naive-strategy.go

```diff
@@ -294,18 +297,17 @@ func (st NaiveStrategy) UnrelayedAcknowledgements(src, dst *ProvableChain, sh Sy
 }
 
 // TODO add packet-timeout support
-func relayPackets(chain *ProvableChain, seqs []uint64, sh SyncHeadersI, sender sdk.AccAddress) ([]sdk.Msg, error) {
+func collectPackets(ctx QueryContext, chain *ProvableChain, seqs []uint64, sender sdk.AccAddress) ([]sdk.Msg, error) {
```

Comment:
`sender` can be mistaken for a packet sender.
Shouldn't it be renamed to `signer` or `txSender`?

Reply:
fixed at https://github.com/hyperledger-labs/yui-relayer/pull/83/commits/6881057cd3b323e434e5660e9e8a208ff4769388